git_machete/tests/functional/test_machete.py

```diff
@@ -672,3 +680,682 @@ def test_help(self) -> None:
                         self.fail(f'Unexpected exception raised: {e}')
                     else:
                         self.fail('SystemExit expected but not raised')
+
+    def test_go_up(self) -> None:
+        """Verify behaviour of a 'git machete go up' command.
+
+        Verify that 'git machete go up' performs 'git checkout' to the
+        parent/upstream branch of a current branch.
```

Comment:
Nit:
```suggestion
        parent/upstream branch of the current branch.
```
since there's only (at most) one, well-defined current branch

Reply:
Here and in the tests below

Reply:
Accepted